source/server/admin/javascript.md

```diff
@@ -1,18 +1,14 @@
 ### Overview
 
-This file contains JavaScript functionality to periodically fetch JSON stats from
-the Envoy server, and display the top 50 (by default) stats in order of how often
-they've changed since the page was brought up. This can be useful to find potentially
-problematic listeners or clusters or other high-cardinality subsystems in Envoy whose
-activity can be quickly examined for potential problems. The more active a stat, the
-more likely it is to reflect behavior of note.
+This describes the flow used for developing, testing, and serving JavaScript,
+which is used to make the admin-site more useful.
 
 ### Serving
 
-By default, this file is converted by the BUILD process and stored as a C++
-absl::string_view constexpr. It is inlined into an HTML response for the stats
-page when the 'active' format is selected. generate_admin_html.sh is used to
-compile HTML and CSS constants into string_view format as well.
+JavaScript, HTML, and CSS files are converted by the BUILD process and a script,
+generate_admin_html.sh. which generates a .cc file with constexpr
```

Comment:
Worth making generate_admin_html.sh a link? (Or if not, backticks around it will at least make it more readable.)

Also backticks around `constexpr absl::string_view`.